k8s/openebs-pre-release-features.yaml

```diff
@@ -911,7 +913,7 @@ spec:
     Create and save list of namespaces to $nss.
     Iterate over each namespace and perform list task
     */ -}}
-    {{- $nss := .Volume.runNamespace | default "" | splitList ", " -}}
+    {{- $nss := .Config.RunNamespace.value | default "" | splitList ", " -}}
```

Comment:
After this change, have you tried -- mayactl volume list?

Reply:
yes. see the comment of https://github.com/openebs/maya/pull/480

Reply:
thanks! got it!